docs/UserGuide.md

```diff
@@ -143,18 +188,33 @@ Format: `s-list`
 
 Finds all drinks where their abbreviated names match the KEYWORD(s).
 
-Format: `s-find KEYWORD [MORE_KEYWORDS] ...`
+Format: `s-find KEYWORD [MORE_KEYWORDS]`
 
-* The search is case-insensitive. e.g `bsbbt` will match `BSBBT`.
-* Only the drink's abbreviated name is searched.
-* Only full words will be matched e.g. `BSB` will not match `BSBBT`.
-* Drinks matching at least one keyword will be returned.
-  e.g. `BSBBT BSBM` will return `BSBBT`, `BSBM`.
+**:information_source: Note:** <br>
+The search is case-insensitive. e.g `bsbbt` will match `BSBBT`.<br>
+Only the drink's abbreviated name is searched.<br>
+Only full words will be matched e.g. `BSB` will not match `BSBBT`. <br>
+Drinks matching at least one keyword will be returned.
+  e.g. `BSBBT BSBM` will return `BSBBT`, `BSBM`.<br>
+
+Example: 
+Let's say you want to find BSBBT and BSBM's sales data, you can follow these instructions:
+
+Steps to find sales data:
+1. Type `s-find BSBBT BSBM` into the _Command Box_.
+1.  Press "Enter" to execute.
+
+Outcome:
+1. The _Result Display_ will show that how many drinks you find.
+
+2. All matching drink's sales data will be listed in the _Main View_.
+
+Before executing:
+![BeforeSalesFind](images/BeforeSalesFind.png)
+
+After executing:
+![SalesFindCommandScreenshot](images/SalesFindResult.png)
```

Comment:
The images cannot be found. Did you remember to upload them?

Reply:
upload the new pictures!